src/pkg/cli/tail.go

```diff
@@ -109,16 +111,16 @@ func ParseTimeOrDuration(str string, now time.Time) (time.Time, error) {
 }
 
 type CancelError struct {
-	Service string
-	Etag    string
-	Last    time.Time
+	Services []string
+	Etag     string
+	Last     time.Time
 	error
 }
 
 func (cerr *CancelError) Error() string {
 	cmd := "tail --since " + cerr.Last.UTC().Format(time.RFC3339Nano)
-	if cerr.Service != "" {
-		cmd += " --name " + cerr.Service
+	if len(cerr.Services) > 0 {
+		cmd += " --name " + strings.Join(cerr.Services, ",")
```

Comment:
I didn't notice the corresponding `Split`

Reply:
Will add.